src/bin/carbonio

```diff
@@ -18,14 +18,26 @@ zal_path="/opt/zextras/lib/ext/carbonio/zal.jar"
 zextras_path="/opt/zextras/lib/ext/carbonio/carbonio.jar"
 carbonio_jars="/opt/zextras/lib/jars/*"
 
+# check for tty presence
+{
+  [[ -t 0 && -t 1 && -n $TERM && $TERM != dumb ]] && which tput &>/dev/null && USE_TPUT=1
+} || USE_TPUT=0
+
+# default columns value
+TTY_COLS = 1
```

Comment:
I will suggest to make it 80 (typically available in a standard terminal window) instead of 1. 

```suggestion
TTY_COLS = 80
```

Reply:
Applied. thanks for the suggestion